whois-commons/src/main/java/net/ripe/db/whois/common/rpsl/DummifierRC.java

```diff
@@ -0,0 +1,70 @@
+package net.ripe.db.whois.common.rpsl;
+
+import com.google.common.collect.Lists;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class DummifierRC extends DummifierCurrent {
```

Comment:
question: shouldn't `mnt-by` also be dummified?...looks like it can be used to retrieve the role object/email details in production

Reply:
Mnt-by refers to a maintainer, and is not personal data in itself. We preserve keys to allow testing (so users can find their data).

We only remove personal data during dummification, and there are limits in production on querying for personal data).